src/arch/riscv/inc/arch/aclint.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Bao Project and Contributors. All rights reserved.
+ */
+
+#ifndef __ACLINT_H__
+#define __ACLINT_H__
```

Comment:
```suggestion
#ifndef ACLINT_H
#define ACLINT_H
```

Underscore started identifiers are reserved by the C standard. I know there are many headers like this in the rest of the code base, but these need to be removed.